src/components/SignUpLink.js

```diff
@@ -39,7 +40,14 @@ export default function SignUpLink({ entry }) {
                 }
             }
             if (!dateLinkPairs.length) {
-                return "No sign-up link available.";
+                return (
+                    <div>
+                        No sign-up link available.
+                        <HelpDialog 
+                            title='No sign-up link available.' 
+                            text="The state website (maimmunizations.org/) is not currently providing a sign-up link for this location."
```

Comment:
How about:
"We were unable to retrieve a sign-up link for this location. It's likely that this location is pulling from a wait list and is not scheduling any new appointments."